### PR TITLE
done

### PR DESCRIPTION
**Pull request by REEMABDULAZIZ**
on a scale from 1 to 5, how complete is this homework?
3 / 5
on a scale from 1 to 5, how comfortable were you with this assignment? 3/5
what was a win you had with this assignment? 
class-flex
what was a challenge you had with this assignment? 
flex-box it was a new challenge .
other stuff on your mind? work with anyone? tell us about it! 
I think  theTime was not enough
yes Had and Abdulaziz

### CODE REVIEW

File: css/main.css [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+.list{
+    background: black;}
+li{
+    color:white;
+    list-style: none;
+    display: inline;
+    text-align: left;
+    text-transform: uppercase;
+}
+section{
```
Comment (JackieCasper) [inline css/main.css:23]: Try using flexbox here to move the inner box to the right

File: css/main.css [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    text-transform: uppercase;
+    height: 45px;
+    width: 200px;
+    background-color: lightgray;
+}
+aside{
+    display: flex;
+
+}
+.div3{
```
Comment (JackieCasper) [inline css/main.css:51]: display flex should be on .div3

**Review by JackieCasper — COMMENTED** [review]
Great work here! 👍 Keep playing around with flexbox, once you get it it can be really helpful!

File: css/main.css (hunk 1 above)
Comment (REEMABDULAZIZ) [reply css/main.css:23]: I will try :)

thank you Jackie for every thing you are a Great Teacher.

**Pull request closed without merging**